lib/index.js

```diff
@@ -29,12 +29,21 @@ function plugin(opts){
     var metadata = metalsmith.metadata();
 
     /**
+     * Clear collections (to prevent multiple additions of the same file when running via metalsmith-browser-sync)
```

Comment:
I think its not specific to browser-sync. https://github.com/segmentio/metalsmith-collections/issues/27